scio-bigquery/src/main/scala/com/spotify/scio/bigquery/BigQueryClient.scala

```diff
@@ -531,7 +530,12 @@ class BigQueryClient private (private val projectId: String,
                   "See https://cloud.google.com/bigquery/docs/reference/standard-sql/")
                 logger.warn(s"Legacy query: `$sqlQuery`")
                 true
-              case Failure(f) => throw f
+              case Failure(f) =>
+                logger.error(
+                  s"Tried both standard and legacy syntax, query `$sqlQuery` failed for both!")
+                logger.error("Standard syntax failed due to:", e)
+                logger.error("Legacy syntax failed due to:")
```

Comment:
Will they be printed to the same output stream though? Depends on logger configuration right?